adage/analyze/tests.py

```diff
@@ -789,30 +789,67 @@ def test_heavy_genes(self):
 
         self.call_non_get_API(uri)  # Test non-get methods too.
 
+    def create_extra_experiments(self):
+        """
+        Generate a few more experiements, one of which is returned
+        (and will be used by test cases later).
```

Comment:
The first experiment in the database was actually created from `ModelsTestCase.experiment_data` in setup(), `exp2` is created here with its `accession` key after `accession` key in `ModelsTestCase.experiment_data`. Three more random experiments are then created by fixtureless. (Because these three experiments are random, their `accession` keys may be before or after accession` key in `ModelsTestCase.experiment_data`. They are created to increase the "noise" and ensure that the two tests that follow can survive the noise. )  Only `exp2` will be returned. Does this make sense?